### DISCUSSION TITLE
Which metric should we use?

### DISCUSSION THREAD

**Discussion by Banboo-glich** [talmolab/sleap#837] [General]
![20220713-233654](https://user-images.githubusercontent.com/65880626/178762826-1de2e0b9-d5cd-4d46-91d8-c181ae98e846.jpg)

I'm trying to track the mice. First of all, please tell me the meaning of VOC. Is VOC the validation data? 

By the way, "VOC with OKS score" is much lower than "VOC with PCK scores". Which metrics should I refer to? 

There is a trade-off between mAP and mAR. Therefore, I wanted to raise the mAP, so in this case I adjusted the sigma to a lower level. Is this the right way to do it?

**Comment by talmo** [comment]
Hi @Banboo-glich,

The VOC metrics relate to the dataset-level mAP/mAR metrics. These can technically be computed with any instance-level error metric, such as OKS (recommended) or PCK (used in other works, so we provide it for comparison).

The OKS is a bit more principled and we recommend checking out [this paper](https://arxiv.org/abs/1707.05388) for more info on how it's defined.

Tuning sigma can have different effects depending on your data. A lower sigma will increase spatial precision and reduce false positives (spurious predictions), at the cost of more false negatives (missing predictions) and potentially a more difficult time training.

**Comment by Banboo-glich** [comment]
Thank you for your response.

It seems that it is customarily appropriate to refer to OKS as the main...

Immediately, I will read the treatise!
